src/NServiceBus.AcceptanceTests/Correlation/When_sending_with_no_correlation_id.cs

```diff
@@ -35,15 +35,13 @@ public CorrelationEndpoint()
 
             public class MyResponseHandler : IHandleMessages<MyRequest>
             {
-                public Context Context { get; set; }
+                public Context TestContext { get; set; }
```

Comment:
In the past we discussed that the `IMessageHandlerContext` becomes a sort of a locator for core provided services. From the perspective of the acceptance tests the `ScenarioContext` is a provided service. I think it would be good to start into that direction in another PR and make the `IMessageHandlerContext` extendable and the ATT framework can be the first user of this API. This makes the container bindings for `ScenarioContext` no longer necessary.